lang/ja.json

```
@@ -719,19 +714,19 @@
       "condition": "プラネタリウムでCelestialエフェクトが入る部屋に入る"
     },
     "hospital_event": {
-      "name": "Patient 001",
-      "description": "This happened to my buddy Eric",
-      "condition": "View the event in the Hospital"
+      "name": "患者番号001",
+      "description": "相棒のエリックに起こった出来事",
+      "condition": "病院内のイベントを見る"
     }
   },
   "someday": {
     "alien": {
-      "name": "Space Party!",
-      "condition": "Visit the giant alien"
+      "name": "スペースパーティー！",
+      "condition": "宇宙船に到達する"
     },
     "dance_flood": {
-      "name": "Dance Flood",
-      "condition": "Flip the underwater switch at the Dance Flood"
+      "name": " クラブハウス",
```

```suggestion
      "name": "クラブハウス",
```
Be careful to avoid extra spaces